examples/js/animation/CCDIKSolver.js

```diff
@@ -45,9 +45,9 @@
 	class CCDIKSolver {
 
 		/**
-	 * @param {THREE.SkinnedMesh} mesh
-	 * @param {Array<Object>} iks
-	 */
+   * @param {THREE.SkinnedMesh} mesh
```

Comment:
@marcofugaro Do you know why these line are changed? Does this not happen if I manually let eslint format the file.

Reply:
@Mugen87 so basically eslint does not touch indentation in comment blocks.
In the `npm run build-examples` script there is babel who rewrites everything and converts tabs to spaces, and then eslint can't convert them back.

Running eslint from the editor, it leaves the indentation in the comment blocks as is, so with tabs.

Reply:
Okay. I think this is a minor issue and can be ignored. The canonical files are in `examples/jsm` and only they have to "look good". For `examples/js` it's sufficient that the examples are functional.